src/hsl.rs

```diff
@@ -65,14 +65,16 @@ impl Hsl {
 
 impl From<LinearRgb> for Hsl {
     fn from(lrgb: LinearRgb) -> Self {
-        let hsl = lrgb
-            .data
-            .iter()
-            .map(|pix| lrgb_to_hsl(*pix))
-            .collect::<Vec<_>>();
+        let mut data = lrgb.data;
+        for pix in &mut data {
+            let hsl = lrgb_to_hsl(*pix);
```

Comment:
Is it possible to do this in one step with `*pix = lrgb_to_hsl(*pix)`?

Reply:
Oh, yes I didn't see that. Good idea